packages/front-end/pages/attributes.tsx

```diff
@@ -69,29 +69,21 @@ const FeatureAttributesPage = (): React.ReactElement => {
               className="dropdown-item"
               onClick={async (e) => {
                 e.preventDefault();
-
-                // update attributes as they render in the view (do not reorder after changing archived state)
-                setAttributesForView(
-                  attributesForView.map((attribute) =>
-                    attribute.property === v.property
-                      ? { ...attribute, archived: !v.archived }
-                      : attribute
-                  )
-                );
-
-                // update SDK attributes while preserving original order
-                attributeSchema = attributeSchema.map((attribute) =>
-                  attribute.property === v.property
-                    ? { ...attribute, archived: !v.archived }
-                    : attribute
-                );
-                await apiCall(`/organization`, {
+                await apiCall<{
+                  res: number;
+                }>(`/attribute/${v.property}`, {
                   method: "PUT",
                   body: JSON.stringify({
-                    settings: { attributeSchema },
+                    property: v.property,
+                    datatype: v.datatype,
+                    projects: v.projects,
+                    format: v.format,
+                    enum: v.enum,
+                    hashAttribute: v.hashAttribute,
+                    archived: !v.archived,
```

Comment:
This is prone to breaking in the future without being caught by CI - for example, if we add a new required property to the attribute schema and forget to update this.

Adding typing here will make it so this will be caught by Typescript at compile time.

```ts
const newAttr: SDKAttribute = {
  // ...
};

//...
body: JSON.stringify(newAttr)
```